src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ExcelAdaptorServiceImpl.java

```diff
@@ -56,6 +60,12 @@
 
     private FormulaEvaluator evaluator;
 
+    @Autowired
+    ValidationServiceFacadeImpl validationServiceFacade;
```

Comment:
Can we use the interface IValidationService instead of implementation class ?

Reply:
done